### PR TITLE
First version

### PR DESCRIPTION
**Pull request by elisabettai**
New repo and code for jupyter-neuron (now it becomes dy-sidecar-powered).

### CODE REVIEW

File: kernels/python-neuron/requirements.in [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# math-base
+h5py
+matplotlib
+numpy
+plotly
+scikit-learn
```
[the remark this answers is not in the document]
Comment (elisabettai) [reply kernels/python-neuron/requirements.in:6]: As far as I understand, requirements.txt is generated when creating the virtualenv inside the container, in the [Dockerfile here](https://github.com/ITISFoundation/jupyter-neuron/pull/1/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R69). Am I misunderstanding?

Should I add a command to create requirements.txt in this repo as well, so we have easy access to the specific versions of the packages installed?

File: .osparc/jupyter-neuron/metadata.yml [hunk 2]
```diff
@@ -1,28 +1,22 @@
-name: JupyterLab Math (Python+Octave)
+name: JupyterLab NEURON
 thumbnail: https://upload.wikimedia.org/wikipedia/commons/thumb/3/38/Jupyter_logo.svg/1024px-Jupyter_logo.svg.png
 description:
   "[JupyterLab](https://jupyter.org/) with **math** kernels in [octave](https://www.gnu.org/software/octave/index)
```
Comment (sanderegg) [inline .osparc/jupyter-neuron/metadata.yml:4]: is the octave part correct?

Comment (sanderegg) [inline .osparc/jupyter-neuron/metadata.yml:4]: is the octave part correct?

Comment (sanderegg) [inline .osparc/jupyter-neuron/metadata.yml:4]: is the octave part correct?

File: .osparc/jupyter-neuron/metadata.yml [hunk 3]
```diff
@@ -1,28 +1,22 @@
...
+    email: iavarone@itis.swiss
     affiliation: IT'IS Foundation
-  - name: Sylvain Andereregg
-    email: anderegg@itis.swiss
-    affiliation: IT'IS Foundation
-  - name: Sylvain Reboux
-    email: reboux@zmt.swiss
-    affiliation: Zurich MedTech
-contact: crespo@itis.swiss
+contact: iavarone@itis.swiss
```
Comment (sanderegg) [inline .osparc/jupyter-neuron/metadata.yml:19]: you might want to put your anonymised github address here?

File: Dockerfile [hunk 4]
```diff
@@ -13,11 +13,10 @@ USER root
 
 RUN apt-get update && \
   apt-get install -y --no-install-recommends \
-  gfortran \
   ffmpeg \
-  make \
   dvipng \
   gosu \
+  make \
   octave \
```
Comment (sanderegg) [inline Dockerfile:20]: octave still needed?

**Review by sanderegg — APPROVED** [review]

File: .osparc/jupyter-neuron/metadata.yml (hunk 2 above)
Comment (elisabettai) [reply .osparc/jupyter-neuron/metadata.yml:4]: Yes. We still have the Octave kernel (as in the previous version of jupyter-neuron).

File: .osparc/jupyter-neuron/metadata.yml (hunk 3 above)
Comment (elisabettai) [reply .osparc/jupyter-neuron/metadata.yml]: Good idea! Thanks for the tip!

File: Dockerfile (hunk 4 above)
Comment (elisabettai) [reply Dockerfile:20]: Yes. Still keeping the Octave kernel.

**Pull request merged**